addusers.py

```diff
@@ -293,7 +293,12 @@ def parse_rows(rows, select_user=None):
                     row=idx, name=entry[15],
                     contact_name=user.first_name + " " + user.last_name,
                     contact_email=user.email,
-                    description='org: {}, contact_name: {}, contact_email: {}, description: {}'.format(entry[8], user.first_name + " " + user.last_name, user.email, entry[16]),
+                    description='org: {},contact_name: {},contact_email: {},\
+                    description: {}'.format(
```

Comment:
This is going to put many spaces in your string.  You can put the `\` outside the quotes:    
```
my_long_string = "This is the first line of a long string. " \
                 "This is the second line."
```
This is another way to write the same thing:   
 ```
my_long_string = ("This is the first line of a long string. "
                   "This is the second line.")
```
Both of these will print `This is the first line of a long string.  This is the second line.`  You can do .format() at the end of either version and either is fine in terms of PEP8 etc.  Although it's good to be consistent within a file about which one to use, the second version is used elsewhere in this file.